src/relay.js

```diff
@@ -0,0 +1,32 @@
+import {fromGlobalId, connectionFromArray, nodeDefinitions} from 'graphql-relay';
+
+export function idFetcher(sequelize) {
+  return globalId => {
+    let {type, id} = fromGlobalId(globalId);
+    const models = Object.keys(sequelize.models);
+    type = type.toLowerCase();
```

Comment:
In that case the model names should be cast to lower case aswell, otherwise you're matching possible uppercase model names against a forced lowercase type.